### ISSUE TITLE
Parser fails on schema-qualified `sequence name` in GENERATED ALWAYS AS IDENTITY

### ISSUE THREAD

**Issue by sumanthratna** [sbdchd/squawk#1249]
squawk 2.60.0 fails to parse `CREATE TABLE` statements whose `GENERATED ALWAYS AS IDENTITY` column uses a schema-qualified `sequence name`. Postgres 16 documents this as valid syntax (the `SEQUENCE NAME sequence_name` clause may be schema-qualified per the `CREATE TABLE` grammar).

## Repro

Unqualified sequence name — parses fine, 0 issues:

```sql
CREATE TABLE "orchestration"."foo" (
	"id" bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY (sequence name "foo_id_seq" INCREMENT BY 1 MINVALUE 1 MAXVALUE 9223372036854775807 START WITH 1 CACHE 1)
);
```

Schema-qualified sequence name — fails to parse:

```sql
CREATE TABLE "orchestration"."foo" (
	"id" bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY (sequence name "orchestration"."foo_id_seq" INCREMENT BY 1 MINVALUE 1 MAXVALUE 9223372036854775807 START WITH 1 CACHE 1)
);
```

Running `squawk` against the second file produces a cascade of `syntax-error` diagnostics anchored at the `.` in `"orchestration"."foo_id_seq"` (`expected R_PAREN`, `expected command, found DOT`, `expected command, found IDENT`, etc.), instead of parsing successfully.

## Why this matters

Tools that schema-qualify identity sequence names whenever the owning table isn't in the `public` schema (e.g. `drizzle-kit generate`) hit this on every migration for such a table. `squawk` then returns pure parser noise instead of real lint results, and there's no way to route around it: `--exclude`/`excluded_rules` reject `syntax-error` as an invalid rule name (`invalid rule name syntax-error`), since it's a parse failure rather than a lint rule.

## Environment

- squawk 2.60.0 (npm)
- macOS

**Comment by sbdchd** [comment]
Fixed via: https://github.com/sbdchd/squawk/pull/1246

**Comment by sbdchd** [comment]
Fixed in v2.61.0 

https://play.squawkhq.com/#code/MIJQogggKmAEUQEIBk4CID2AnAxgCwFMBnAFywEMSBLDAOzQDo0AzDDNWACgCgBINKgBMOAIyoBzKrRKwACiACSAWQggAmrADSYDQHEwAOTAhoYACKwIyAOoQ1AZUuOFZw1AVQNnIgQCOAVwJaHAJYWnIAW1DMXEJSCmo6RhY2AH0hVJ9fDgUDUDAlN1hEDQBGWCVcgDUrAFU4cpUADRrketgATgAmLoBmXoB2LoAGXoA2AA4AVgAWAYGpieGB2HsEEChYaw8ACVhy4AhgHYaASm5TgG4gA

Thank you for the report!

**Comment by sumanthratna** [comment]
@sbdchd Thanks for prepping a new release so quickly!
